WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1678,21 +1743,25 @@ extension NotificationsViewController: UIViewControllerTransitioningDelegate {
 
             UserDefaults.standard.notificationPrimerAlertWasDisplayed = true
 
-            let alert = FancyAlertViewController.makeNotificationPrimerAlertController { (controller) in
-                InteractiveNotificationsManager.shared.requestAuthorization {
-                    DispatchQueue.main.async {
-                        controller.dismiss(animated: true)
-                    }
-                }
-            }
+            let alert = alertController
             alert.modalPresentationStyle = .custom
             alert.transitioningDelegate = self
             self?.tabBarController?.present(alert, animated: true)
         }
     }
 
+    private func showSecondNotificationsAlertIfNeeded() {
+        guard userDefaults.secondNotificationsAlertCount >= Constants.secondNotificationsAlertThreshold else {
+            return
+        }
+        showSecondNotificationAlert()
+        userDefaults.secondNotificationsAlertCount = Constants.secondNotificationsAlertDisabled
+    }
+
     private enum Constants {
         static let inlineTabAccessCount = 6
         static let displayAlertDelay = 0.2
+        static let secondNotificationsAlertThreshold = 10
```

Comment:
Could we add one small comment to clarify what this is used to count?

Reply:
👍